src/transformers/tokenization_utils_fast.py

```diff
@@ -172,10 +172,12 @@ def __init__(self, *args, **kwargs):
         # allows converting a slow -> fast, non-legacy: if the `tokenizer.json` does not have all the added tokens
         # uses the information stored in `added_tokens_decoder`.
         # this is costly for fast tokenizers as we re-compute the regex again. But not all tokens are added tokens
+        # Use hash to speed up the very slow operation `token not in added_tokens_decoder`.
+        added_tokens_decoder_hash = {hash(token) for token in self.added_tokens_decoder}
```

Comment:
there is a very very tiny chance of hash collision. Do we want to address that possibility?

Reply:
no 😉